<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ogrenci_islemleri</class>
 <widget class="QWidget" name="ogrenci_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_no">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_soyad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_no">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ad">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_soyad">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yeni">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sil">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>470</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_guncelle">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_tumogr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>421</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tumogr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
